src/dali/plugin/input/csv/blockfi.py

```diff
@@ -50,11 +55,12 @@ def __init__(
         self,
         account_holder: str,
         csv_file: str,
```

Comment:
Rename 'csv_file' to 'transaction_csv_file', for clarity and simmetry with 'trade_csv_file'.